services/core-api/app/api/now_applications/transmogrify_now.py

```diff
@@ -149,6 +150,57 @@ def _transmogrify_contacts(now_app, now_sub, mms_now_sub):
     return
 
 
+def _transmogrify_clients(now_app, now_sub, mms_now_sub):
+    agent = now_sub.submitter
+    emailValidator = re.compile(r'[^@]+@[^@]+\.[^@]+')
+    now_party_appt = None
+    if agent.type == 'Individual' and agent.ind_lastname and agent.ind_firstname and agent.ind_phonenumber:
+        now_party = Party(
+            party_name=agent.ind_lastname,
+            first_name=agent.ind_firstname,
+            party_type_code='PER',
+            phone_no=agent.ind_phonenumber[:3] + "-" + agent.ind_phonenumber[3:6] + "-" +
+            agent.ind_phonenumber[6:],
+            email=agent.email if agent.email and emailValidator.match(agent.email) else None,
+        )
+        now_party_appt_type = MinePartyAppointmentType.find_by_mine_party_appt_type_code('AGT')
+        now_party_appt = app_models.NOWPartyAppointment(
+            mine_party_appt_type_code=now_party_appt_type.mine_party_appt_type_code,
+            mine_party_appt_type=now_party_appt_type,
+            party=now_party)
+    if agent.type == 'Organization' and agent.contacttype and agent.org_legalname and agent.dayphonenumber:
+        now_party = Party(
+            party_name=agent.org_legalname,
+            party_type_code='ORG',
+            phone_no=agent.dayphonenumber[:3] + "-" + agent.dayphonenumber[3:6] + "-" +
+            agent.dayphonenumber[6:],
+            phone_ext=agent.dayphonenumberext,
+            email=agent.email if agent.email and emailValidator.match(agent.email) else None,
+        )
+        now_party_appt_type = MinePartyAppointmentType.find_by_mine_party_appt_type_code('AGT')
+        now_party_appt = app_models.NOWPartyAppointment(
+            mine_party_appt_type_code=now_party_appt_type.mine_party_appt_type_code,
+            mine_party_appt_type=now_party_appt_type,
+            party=now_party)
+
+    if now_party_appt:
+        validPostalCode = re.compile(r"\s*([a-zA-Z]\s*\d\s*){3}$")
```

Comment:
We have a different postal code regEx on the FE, should they match?
```
/^[A-Za-z]\d[A-Za-z][ -]?\d[A-Za-z]\d$/
```

Reply:
that is the way both party imports work